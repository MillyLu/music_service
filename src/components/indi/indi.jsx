
/* eslint-disable react/jsx-no-bind */

import { useContext} from 'react';
import  Navigation  from "../navigation/nav";
import { ThemeContext } from '../../ThemeProvider';
import * as Styled from './styles'
import { Footer } from '../footer/footer';
import { MainIndiCenterBlock } from '../indi-centerBlock/mainIndiCenterBlock';







export function PlaylistContainer() {
    const {theme} = useContext(ThemeContext);
    return(
        <Styled.PlaylistContainer theme={theme}>
            <Styled.Main theme={theme}>
                <Navigation />
                <MainIndiCenterBlock />
            </Styled.Main>
            <Footer />
        </Styled.PlaylistContainer>
    )
}