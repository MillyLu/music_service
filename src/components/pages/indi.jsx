import  {PlaylistContainer}  from "../indi/indi";

export function Indi() {
    return(
        <PlaylistContainer />
    )
}