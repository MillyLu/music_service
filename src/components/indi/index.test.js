/* eslint-disable no-undef */
/* eslint-disable no-unused-expressions */
import * as ShallowRenderer from 'react-shallow-renderer';
import { PlaylistContainer } from './indi';


describe('<PlaylistContainer/>', () => {
    it('should render successfully and match the snapshot', () => {
        const shallowRenderer = ShallowRenderer.createRenderer();
        const renderer = shallowRenderer;
        renderer.render(<PlaylistContainer />);
        const result = renderer.getRenderOutput();
        expect(result).toMatchSnapshot();
    })
})