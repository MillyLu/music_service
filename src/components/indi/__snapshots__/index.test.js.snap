// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PlaylistContainer/> should render successfully and match the snapshot 1`] = `
<styled.div
  theme={0}
>
  <styled.div
    theme={0}
  >
    <Navigation />
    <MainIndiCenterBlock />
  </styled.div>
  <Footer />
</styled.div>
`;
